### PR TITLE
Improve description of coordinate system

### LINKED ISSUE #5: Alignment / Scaling / Spacing
As discussed in #1, how should we handle alignment and scaling?

My suggestion would be to use the scale between 0-100 for the distance between baseline and cap height:

terminology

The layout engine would put y=0 on the baseline and scale the fonts according to the chosen font size so that y=100 is at the cap height.

Spacing could be done in a fixed way for now, so simply get the width of the character, left-align it and add some space on the right before placing the next character. Some simple kerning support could be added later.

I'm not sure how to handle the space character though. It could be derived from the width of a specific glyph, e.g. the capital A? Not sure if this is something that is being done by font layout engines or not.

Discussion:

- [ ] Fixed scale for height?
- [ ] Spacing between characters?
- [ ] Width of the space character?

### PR DESCRIPTION
**Pull request by ubruhin**
This very important thing was not really good explained until now...

Closes #5.

### CODE REVIEW

File: SPECIFICATION.md [hunk 1]
```diff
@@ -101,17 +101,23 @@ the number may be omitted. If the integer part is zero and there is a fractional
 part, the zero may be omitted (e.g. `-.5` instead of `-0.5`).
 
 The coordinate system starts at the lower left corner. Positive X coordinates
-point to the right. Positive Y coordinates point to the top. Negative
-coordinates are not allowed.
+point to the right. Positive Y coordinates point to the top. A value of 9 means
+100% of the font height (i.e. the coordinate system is scaled with factor 9).
```
Comment (dbrgn) [inline SPECIFICATION.md]: "Font height" is ambiguous. Maybe use the term "cap height"? See https://github.com/fontobene/fontobene/issues/5

We should probably also include that illustration in the spec. We have permission to do so.

File: SPECIFICATION.md [hunk 2]
```diff
@@ -101,17 +101,23 @@ the number may be omitted. If the integer part is zero and there is a fractional
...
           Y
-    (0,Y) +
+    (0,9) +
           |
           |
           +-----+ X
-      (0,0)   (X,0)
+      (0,0)   (9,0)
+
+Typical glyphs use the X axis (Y=0) as the baseline and are drawn inside the
```
Comment (dbrgn) [inline SPECIFICATION.md]: I'd write "drawn mostly inside" instead of "drawn inside".

File: SPECIFICATION.md [hunk 3]
```diff
@@ -101,17 +101,23 @@ the number may be omitted. If the integer part is zero and there is a fractional
...
+area (0,0)..(9,9). Coordinates outside this area are allowed, e.g. for ascenders
+and descenders. And typical glyphs are left aligned to the Y axis (X=0).
```
Comment (dbrgn) [inline SPECIFICATION.md]: I'd remove the "And".

File: SPECIFICATION.md [hunk 4]
```diff
@@ -101,17 +101,23 @@ the number may be omitted. If the integer part is zero and there is a fractional
...
+Otherwise it influences letter spacing, see section "Letter Spacing" for details.
```
Comment (dbrgn) [inline SPECIFICATION.md]: Maybe write "Glyphs that are not aligned to the Y axis have an impact on letter spacing, (...)" instead?

**Review by dbrgn — CHANGES_REQUESTED** [review]

**Comment by dbrgn** [comment]
By the way, would be great if the changes were detailed enough to be able to close #5.

**Comment by ubruhin** [comment]
Added image and moved coordinate system specification to a separate section to not clutter up the polyline section too much -> [rendered](https://github.com/fontobene/fontobene/blob/40bf0ab74ba3efb1bf25f4b62c2c6fb86f44dd5b/SPECIFICATION.md)

**Review by dbrgn — APPROVED** [review]
Great, thanks! Feel free to merge if you think this is ready.

**Pull request merged**